kernel/arch/ia32/include/arch/boot/boot.h

```diff
@@ -44,7 +44,7 @@
 #ifdef CONFIG_SMP
 
 /* This is only a symbol so the type is dummy. Obtain the value using &. */
-extern int _hardcoded_unmapped_size;
+extern errno_t _hardcoded_unmapped_size;
```

Comment:
Again, there is no error information in this. We only care for the address of this symbol.